homeassistant/components/devolo_home_control/__init__.py

```diff
@@ -76,6 +80,22 @@ def shutdown(event):
     return True
 
 
+async def async_migrate_entry(hass, entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s", entry.version)
+
+    if entry.version == 1:
+        mydevolo = _mydevolo(entry.data)
+        new = {**entry.data}
+        new[CONF_UNIQUE_ID] = await hass.async_add_executor_job(mydevolo.uuid)
```

Comment:
The config entry unique_id is an attribute of the config entry. It's not part of the config entry data.